erpnext/crm/doctype/lead/lead.js

```diff
@@ -16,6 +16,7 @@ erpnext.LeadController = frappe.ui.form.Controller.extend({
 	},
 
 	onload: function () {
+		set_field_options("type", [" ", "Client", "Channel Partner", "Consultant", "Investor"]);
```

Comment:
Why not add these in the JSON? You won't be able to create leads programmatically or using the Data Import tool this way